inputs/wysiwyg.vue

```diff
@@ -146,7 +146,7 @@
   }
 
   // all phrases should have some sort of highlight, so users can
-  // reason about them in edit mode. for now, we're doing different colors
+  // reason about them in edit mode. for now, we are doing different colors
```

Comment:
this apostrophe was fucking up syntax highlighting :(

Reply:
get you a text editor that can do both 😛